Weather Man/utils.py

```diff
@@ -0,0 +1,86 @@
+from constants import colors
+from datetime import datetime
+from weather_data import WeatherData
+from constants import EXPECTED_NUMBER_OF_ARGS
+import pandas as pd
+import sys
+import os
+
+
+def validate_args(args):
+    if len(args) != EXPECTED_NUMBER_OF_ARGS:
+        print("Usage: weatherman.py /path/to/files-dir -e 2002 or -a 2002/3 or -c 2002/12")
+        sys.exit(1)
+
```

Comment:
Docstring not written, Also should this be in some class that processes input when coming from main function